addons/units/functions/fnc_onAltLoad.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+
+params ["_ctrl"];
+
+private _alt = (ctrlParent _ctrl) ctrlCreate [QGVAR(alt), -1];
+
+private _units = (vehicle ace_player) call FUNC(altUnits);
+private _altInfo = [_units select 0] call FUNC(altInfo);
+
+systemChat "alt time";
+
+((ctrlParent _alt) displayCtrl 1005) ctrlSetText (_altInfo select 0);
+GVAR(altRatio) = _altInfo select 1;
```

Comment:
`params` would still be benefitial, even though you have two more locals. Same below.